### DISCUSSION TITLE
How to Display qr code for scanning on desktop mode

### DISCUSSION THREAD

**Discussion by skapezempire** [google/model-viewer#4702] [Q&A] [answered]
Hello world, I need assistance with this;

How to display a qr code when user clicks on the "view in space" button on a desktop, so the user can scan with their mobile device for the Ar experience.

Here is an example in the image below.

![Screenshot 2024-03-06 163632.png](https://github.com/google/model-viewer/assets/18077781/5f7f8c90-a3aa-4b0a-8109-a5c5e7758cf3)

Thank you

**Comment by danilokrulj** [comment, marked as answer]
I use a library like this one

 https://github.com/davidshimjs/qrcodejs 
 
 to generate a QR code leading to the page where you hosted your model and pass the parameters that change through the url. 
 
 If your model doesn't change you can just generate a QR code once and embed it inside a div element which you would show by toggling the display settings on it when you click the "View in AR" button.

**Reply by skapezempire** [reply]
Thanks for sharing.
